src/Microsoft.Azure.SignalR.Protocols/Models/ServiceModelProtocol.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Buffers;
+using MessagePack;
+
+namespace Microsoft.Azure.SignalR.Protocol;
+
+#nullable enable
+
+public class ServiceModelProtocol
```

Comment:
is internal enough?

Reply:
Runtime needs this class to serialize model.